bedrock/settings/base.py

```diff
@@ -366,11 +366,20 @@ def set_whitenoise_headers(headers, path, url):
     }
 }
 
+
+def get_app_name(hostname):
+    if hostname.startswith('bedrock-'):
```

Comment:
Maybe a comment here as to the naming convention/expected possible values? (You know, for the next time our infra changes :trollface:)